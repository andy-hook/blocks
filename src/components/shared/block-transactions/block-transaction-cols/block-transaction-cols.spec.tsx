import React from "react"
import BlockTransactionCols from "./block-transaction-cols"
import { renderWithTheme } from "@test-utils"

jest.mock("react-truncate-string")

describe("<BlockTransactionCols />", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme(
      "dark",
      <BlockTransactionCols
        block="string"
        hash="string"
        fromTo="string"
        value="string"
      />
    )
    expect(tree).toMatchSnapshot()
  })
})
