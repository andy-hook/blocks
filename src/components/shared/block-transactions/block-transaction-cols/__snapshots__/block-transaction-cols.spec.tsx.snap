// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlockTransactionCols /> renders correctly 1`] = `
<tr
  className="sc-bdVaJa bDWFJH"
>
  <td
    className="sc-bwzfXH sc-htpNat KtFsv"
  >
    string
  </td>
  <td
    className="sc-bwzfXH sc-bxivhb bUZGRr"
  >
    string
  </td>
  <td
    className="sc-bwzfXH sc-ifAKCX cFlEyZ"
  >
    string
  </td>
  <td
    className="sc-bwzfXH sc-EHOje jvzENE"
  >
    string
  </td>
</tr>
`;
